hack/test_controllers/rctest1.json

```diff
@@ -0,0 +1,32 @@
+{
+   "kind":"ReplicationController",
```

Comment:
I suppose I could create the rc from a string and parse through it in bash to change names, but can I get away with this? If my reviewers feel strongly I'll remove this test and do it in a follow up pr, but it's in hack/test_controllers so meh?